Source/HelixToolkit.Wpf/Helpers/CameraHelper.cs

```diff
@@ -657,7 +657,9 @@ public static Matrix3D GetInverseTransform(this Camera camera, double aspectRati
         /// </param>
         public static void ZoomExtents(this ProjectionCamera camera, Viewport3D viewport, double animationTime = 0)
         {
-            var bounds = Visual3DHelper.FindBounds(viewport.Children);
+            var bounds =
+                viewport.Children.FindBounds(
+                    new List<Type> { typeof(BillboardTextGroupVisual3D), typeof(BillboardTextVisual3D) });

```

Comment:
not open for extension! If you create a new type of screen space visual in your application (not in the Helix Toolkit library), it cannot be ignored...